spectrocloud/schemas/cluster_node.go

```diff
@@ -0,0 +1,28 @@
+package schemas
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func NodeSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"node_id": {
+					Type:        schema.TypeString,
+					Required:    true,
+					Description: "The node_id of the node.",
```

Comment:
Please provide an example how the node id may look like or where can be taken from i.e. API reference.